recipes/bio-embeddings/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "bio-embeddings" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bio_embeddings-{{ version }}.tar.gz
+  sha256: a82d27d895d971bb30339071440f9c5e75d31cf9f4a8a600d17efe7bea0e8615
+
+build:
+  entry_points:
+    - bio_embeddings = bio_embeddings.utilities.cli:main
+  skip: true  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+    - poetry >=1.1.0
+  run:
+    - appdirs >=1.4.4,<2.0.0
+    - atomicwrites >=1.4.0,<2.0.0
+    - biopython >=1.79,<2.0
+    - gensim >=3.8.2,<4.0.0
+    - h5py >=3.2.1,<4.0.0
+    - humanize >=3.2.0,<4.0.0
+    - importlib_metadata >=4.6.1,<5.0.0
+    - lock >=2018.3.25,<2019.0.0
+    - matplotlib-base >=3.2.1,<4.0.0
+    - numpy >=1.18.3,<2.0.0
+    - pandas >=1.2.0,<2.0.0
+    - plotly >=5.1.0,<6.0.0
+    - python
+    - python-slugify >=5.0.2,<6.0.0
+    - ruamel.yaml >=0.17.10,<0.18.0
+    - scikit-learn >=0.24.0,<0.25.0
+    - scipy >=1.4.1,<2.0.0
+    - pytorch >=1.8.0,<=1.10.0
+    - tqdm >=4.45.0,<5.0.0
+    - umap-learn >=0.5.1,<0.6.0
+
+test:
+  imports:
+    - bio_embeddings
+  commands:
+    - pip check
+    - bio_embeddings --help
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/bio-embeddings/
```

Comment:
The dev url is https://github.com/sacdallago/bio_embeddings ?

Reply:
yes